RickLeinecker/Fortuna
# invalidate token if logged in twice

**Description**
Invalidates tokens from older sessions to enforce single user logins.

**Resolves These Issues**
Resolves #477 

**Type of change**
Check options that are relevant.


**Steps to Verify Functionality**
Make a bulleted/numbered list of steps to verify this feature/bugfix. Include screenshots if necessary.
1) Login user
2) make an authenticated api call and see if it is successful (it should be)
3) login in a new window and redo step 2
4) go back to the first login window and try to make an authenticated api call and it should no longer work.

**Final Checklist:**
Go down the list and check them off.
